tests/spec/core/structure-spec.js

```diff
@@ -72,26 +83,37 @@ describe("Core - Structure", () => {
     };
     ops.config.maxTocLevel = 4;
     const doc = await makeRSDoc(ops);
-    const $toc = $("#toc", doc);
-    expect($toc.find("h2").text()).toEqual("Table of Contents");
-    expect($toc.find("> ol > li").length).toEqual(3);
-    expect($toc.find("li").length).toEqual(11);
+    const toc = doc.getElementById("toc");
+    expect(toc.querySelector("h2").textContent).toEqual("Table of Contents");
+    const heirLI = Array.prototype.slice
+      .call(toc.children)
+      .reduce((result, child) => {
+        if (child.nodeName !== "OL") return result;
+        const childrenLi = [...child.children].filter(
+          element => element.nodeName === "LI"
+        );
+        return [...result, ...childrenLi];
+      }, []);
+    expect(heirLI.length).toEqual(3);
```

Comment:
I think we can do this with querySelector itself.
``` js
expect(doc.querySelectorAll("#toc > ol > li").length).toEqual(3);
```

Reply:
My bad, wasn't working in first try. Thanks :)